VulnerableApplication/Global.asax.cs

```diff
@@ -17,5 +17,6 @@ protected void Application_Start()
             RouteConfig.RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
         }
+        public const string Password = "abc";
     }
```

Comment:
**CODE_SMELL**
> Change this constant to a 'static' read-only property.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AW1FS3clVuKkC1XNKm4s&pullRequest=18&resolved=false)

Reply:
/falsepositive

Reply:
The issue is already resolved or confirmed.

Reply:
/falsepositive

Reply:
The issue is already resolved or confirmed.